docs/credits.md

```diff
@@ -1180,6 +1180,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | sane                                              | amasad                                                                    | [https://github.com/amasad/sane](https://github.com/amasad/sane)                                                                                                                                                            | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | Sashimi.Aws.Accounts                              | Sashimi.Aws.Accounts                                                      | [https://github.com/OctopusDeploy/Sashimi/](https://github.com/OctopusDeploy/Sashimi/)                                                                                                                                      | [Apache-2.0](https://licenses.nuget.org/Apache-2.0)                                                                 |
 | Sashimi.Aws.Common                                | Sashimi.Aws.Common                                                        | [https://github.com/OctopusDeploy/Sashimi/](https://github.com/OctopusDeploy/Sashimi/)                                                                                                                                      | [Apache-2.0](https://licenses.nuget.org/Apache-2.0)                                                                 |
+| Sashimi.Azure                                     | Sashimi.Azure                                                             | []()                                                                                                                                                                                                                        | [Undefined]()                                                                                                       |
```

Comment:
Ping @slewis74 - think we might be missing some license details over there (not sure if you already know about this...?)

Reply:
I'm in the process of adding the license details now